packaging_uom/models/product_packaging.py

```diff
@@ -10,10 +9,10 @@ class ProductPackaging(models.Model):
 
     @api.model
     def _default_uom_categ_domain_id(self):
-        uom_id = self.env.context.get("get_uom_categ_from_uom")
```

Comment:
@jhumfer You delete this behaviour but you let the context in view. Could you explain why you changed this ?

Reply:
With the change to version 11, the product.packaging model has a product.product associated instead of product.template, causing the "get_uom_categ_from_uom" context to stop working. 
I have modified it in view as well.